store/postgres/migrations/000020_add_job_dependencies_table.down.sql

```diff
@@ -1 +1,4 @@
-ALTER TABLE job DROP IF EXISTS job_dependency;
\ No newline at end of file
+DROP TABLE IF EXISTS job_dependency;
```

Comment:
why do we drop the table first mbak instead of dropping the index first?

Reply:
Turns out, dropping table also includes dropping the indexes ([ref](https://www.postgresql.org/docs/14/sql-droptable.html)). So, I will remove the drop index instead. @irainia @sbchaos

Reply:
cool, that also works